.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,17 +1,14 @@
 blank_issues_enabled: true
 contact_links:
-  - name: OpenAPI Specification
-    url: https://spec.openapis.org/oas/latest.html
-    about: View the latest version of the specification
-  - name: Getting started documentation
-    url: https://learn.openapis.org
-    about: Learn about the OpenAPI Specification
-  - name: OpenAPI Tooling
-    url: https://tools.openapis.org
-    about: Open-source and commercial tools for OpenAPI
-  - name: OpenAPI Initiative Registry
-    url: https://spec.openapis.org/registry/index.html
-    about: Registry of formats, extension namespaces etc.
```

Comment:
Why omit this one?

Reply:
@ralfhandl I remembered why I took this out- the "official" way to include links to filing issues on the current repository is to make issue templates for them, but GitHub forces those links to be at the top of the list.  Which defeats the entire point of the configuration which is to route people _away_ from filing issues here.

But I realized that it's just a different URL to bypass the issue chooser, and we can use that URL as a "contact" to put local issue-filing at the end.  So now I've done that 🙂